popSimManu.tex

```diff
@@ -158,13 +159,14 @@ \section*{Introduction}
 realistic simulations for population genetic analysis a simple matter of
 choosing pre-implemented models from a community maintained catalog.
 The \stdpopsim catalog currently contains four organisms: humans,
-\emph{D.\ melanogaster}, \emph{A. thaliana}, and \emph{E. coli}. For each
+\emph{D.~melanogaster}, \emph{A.~thaliana}, and \emph{E.~coli}. For each
```

Comment:
latex will treat `. ` like the end of a sentence, with extra space; `~` is a non-breaking space

Reply:
"\ " after the "." avoids that AFAIK, but you're right, ~ is better.